bugsnag-plugin-android-okhttp/src/main/java/com/bugsnag/android/okhttp/BugsnagOkHttpPlugin.kt

```diff
@@ -25,6 +32,7 @@ import java.util.concurrent.ConcurrentHashMap
  * https://square.github.io/okhttp/4.x/okhttp/okhttp3/-response-body/#the-response-body-must-be-closed
  */
 class BugsnagOkHttpPlugin @JvmOverloads constructor(
+    private val delegateEventListener: EventListener? = null,
     internal val timeProvider: () -> Long = { System.currentTimeMillis() }
 ) : Plugin, EventListener() {
```

Comment:
I think it's worth extracting the `EventListener` delegation implementation should probably be extracted into a new `internal` class, so that we can keep the delegation & action plugin implementations separate.